docs/pywbemclicommands.rst

```diff
@@ -535,42 +529,69 @@ The **instance** group defines commands that act on CIM instances including:
   WBEM server. The command build the CIMInstance from the class defined by
   CLASSNAME and the properties defined by the ``--property``\``-p`` option The
   properties are defined as name/value pairs, one property for each instance of
-  the ``--property`` option. Since the WBEM server (and pywbem) requires that
-  each property be typed, pywbemtools uses the CIMClass defined by CLASSNAME
-  retrieved from the WBEM server to define the type required to define the
-  CIMProperty.
+  the ``--property`` option.
 
-  For a single property in the new instance this is simply the `--property`` option
-  with the property name and value:
+  The ``--property`` argument value component defines the value of the
+  property. Since the WBEM server (and pywbem) requires that each property be
+  typed, pywbemcli uses the CIMClass defined by CLASSNAME retrieved from the
+  WBEM server to define the type required to define the CIMProperty. The value
+  argument contains just the value itself in the form of a string or numeric
+  value. For numeric values, the creation will fail if the values of the
+  numeric exceeds the range of the CIM type for the property defined in the
+  class (ex. -3 for Uint32).
+
+  See [#fbackslash]_ for information on use of backslashes in formating
+  property argument values:
+
+  For a single property in the new instance this is simply the `--property``
+  option with the property name and value:
 
   .. code-block:: text
 
-    --property <property-name>=<property-value"
+    --property <property-name>=<property-value
+
+    where quotes around the value are only required if the value includes
+    whitespace.
```

Comment:
I think instead of giving an (incomplete) description here, we should have a section "Specifying CIM property and parameter values" that is similar to "Specifying INSTANCENAME" and that defines the format precisely, e.g. by referring to DSP0004 stringValue etc.

Reply:
Good idea. I was thinking about it also.  I added the section and referenced it from both the class and instance invokemethod definitions. NOTE: I put a TODO on this one since the first goal was to replace the short options and I need to do more cleanup on the text in this section.  We have this pr, the sections naming pr and your pywbmbemclicommands.rst pr that all get merged and then we can look at this chapter in more detail

Reply:
DONE with 2 TODOs in the documentation to refactor the text.